flaml/autogen/agentchat/responsive_agent.py

```diff
@@ -262,19 +264,25 @@ def _append_oai_message(self, message: Union[Dict, str], role, conversation_id:
 
         Args:
             message (dict or str): message to be appended to the ChatCompletion conversation.
-            role (str): role of the message, can be "assistant" or "function".
+            role (str): role of the message, can be "assistant", "user" or "function", this will overwrite the `role` in the message only when original role is not "function".
```

Comment:
```suggestion
            role (str): role of the message, can be "assistant", "user" or "function", this will overwrite the "role" in the message only when original role is not "function".
```